.gitignore

```
@@ -13,6 +13,7 @@ build/
 dist/
 Docs/docs/
 Resolvers/
+Ayon_Usd_Resolver_cicd/
```

What about gitignoring `build_conf.yaml`. I like to have it in my repo clone.
```suggestion
Ayon_Usd_Resolver_cicd/
build_conf.yaml
```

---

the conf file can have any name so it won't make much sense to have it like this.